modin/pandas/test/dataframe/test_indexing.py

```diff
@@ -433,6 +433,21 @@ def test_loc_multi_index():
     df_equals(modin_df.loc[modin_df.index[:7]], pandas_df.loc[pandas_df.index[:7]])
 
 
+def test_loc_empty():
+    pandas_df1 = pandas.DataFrame(
+        np.array([i ** 2 for i in range(5)]), columns=["TIME"]
+    )
+    pandas_df2 = pandas.DataFrame(index=pandas_df1.index)
```

Comment:
what's the point of creating a DataFrame if we use only an index from it? Am I missing something?
```suggestion
    pandas_df = pandas.DataFrame(index=range(5))
```

Reply:
You're right, not necessary for the test. I was just reproducing the exact script that was the reported bug.